app/src/main/java/io/appium/uiautomator2/server/AppiumServlet.java

```diff
@@ -103,6 +104,7 @@ private void registerPostHandler() {
         register(postHandler, new TouchDown("/wd/hub/session/:sessionId/touch/down"));
         register(postHandler, new TouchUp("/wd/hub/session/:sessionId/touch/up"));
         register(postHandler, new TouchMove("/wd/hub/session/:sessionId/touch/move"));
+        register(postHandler, new AllowInvisibleElements("/wd/hub/session/:sessionId/appium/device/allowInvisibleElements"));
```

Comment:
Please updating endpoint to `/wd/hub/session/:sessionId/appium/settings` same way as the appium [API](https://github.com/appium/appium-base-driver/blob/master/lib/mjsonwp/routes.js#L430-L432), rename `AllowInvisibleElements` to `UpdateSettings` and add property to [session](https://github.com/appium/appium-uiautomator2-server/blob/8774414fdd6b5b3b846039af12de78c9bd3b2bbd/app/src/main/java/io/appium/uiautomator2/model/Session.java#L15)